pkg/controllers/clusteroperator_controller_test.go

```diff
@@ -228,10 +228,7 @@ var _ = Describe("Component sync controller", func() {
 	var operands []client.Object
 	var watcher mockedWatcher
 
-	externalFeatureGateAccessor := featuregates.NewHardcodedFeatureGateAccess(
-		[]configv1.FeatureGateName{configv1.FeatureGateExternalCloudProvider},
-		nil,
-	)
+	featureGateAccessor := featuregates.NewHardcodedFeatureGateAccess(nil, nil)
```

Comment:
just a question, looking at the tests it seems like we can factor out this variable. what am i missing?

Reply:
The clusteroperator still uses a featuregateaccess underneath to render feature gates for the operand resources, so I had left it in, in case we need to update the cases, but, I think you're right, that would probably be tested elsewhere so we maybe don't need it in the test cases here